docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,120 @@
+with it (like "Structures may be semantically equivalent even though, schematically, they are clearly not"). ODRL
+enables a Provider to be very specific when it comes to the usage of data: What may a consumer do with it? What
+obligations does a consumer incur when accepting the offer?
+
+The first time the Consumer encounters a Policy is when evaluating a Provider's [Catalog](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/catalog/catalog.protocol). These four Offers are
+of `@type` `odrl:Set` which is a [subclass](https://www.w3.org/TR/odrl-model/#policy-set) of `odrl:Policy`.
+All terms can be used interchangeably, Offer is most common. Their content is contained in the
+`odrl:permission`, `odrl:prohibition` and `odrl:obligation` properties. They hold `odrl:Rules` that have to be
+satisfied by the Consumer. There's two ways that can happen.
+
+### Consumer-side: `odrl:Offer` in a `ContractRequestMessage`
+
+Firstly, when transmitting a [`ContractRequestMessage`](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/contract-negotiation/contract.negotiation.protocol#id-2.1-contract-request-message)
+(for example triggered via the EDC's [contract-negotiations API](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/usage/management-api-walkthrough/05_contractnegotiations.md)), the
+Consumer must signify consent to the Offer by including it in the Request and adding the `odrl:target` property on the
+level of an `odrl:Offer`. It is a very, very bad idea to simply copy the next-best Offer from the Catalog and replay it
+to the Provider. Instead, Consumers must perform internal checks which (if any) Offer proposes terms that they want to
+agree to. It is possible (though not usual) to encode pricing information in an `odrl:Offer`. Sending a
+`ContractRequestMessage` echoing the pricing information would represent a legally binding agreement in accordance to
+the conditions proposed by the Provider's Offer. Thus, selecting an offer extended by a Provider company should thus be
+handled with appropriate care.
+
+The Provider will usually accept the ContractRequest only if the extended Offer is semantically equivalent to his own
+as distributed via the Catalog.
+
+### Provider-side: Checking a Consumer's Verifiable Presentation
+
+In addition to checking the Offer's structure (as above), Providers' Connectors have the chance to query the Consumer's
+Verifiable Presentation (VP). A VP is a set of Verifiable Credentials (VCs) that a Consumer may have been issued by a
+trusted third party. In this PR, a Provider may find additional information if a Consumer is eligible to pass a certain
+`odrl:Constraint`.
+
+If, for example, the Consumer tries to negotiate for an Offer that is extended only to interested
+parties from civil society (like an NGO), simply pretending to be an NGO shouldn't be enough. It has to be verified and
```

Comment:
Example should be closer to Catena-X, e.g. Dismantler.

Reply:
As explained - I think there's value in explaining the concepts in the abstract and then narrowing it down to its application in Catena-X

Reply:
abstract makes sense, yes. but different use case imho not.

Reply:
Same, I would go for the MembershipCredential